Libraries/LibWeb/HTML/HTMLTitleElement.cpp

```diff
@@ -4,19 +4,27 @@
  * SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Commit message: please auto-wrap your commit message lines at 72 characters (see CONTRIBUTING.md)

Reply:
yeah I have to do it manually cuz idk how to do that automatically in git / vsc :P